### PR TITLE
#166315332  add a readme file with build/coverage status

### PR DESCRIPTION
**Pull request by Chiagozielam**
#### What does this PR do?
- adds a readme file that displays the status of your build/coverage upon a PR

#### Description of Task to be completed?
- create a `README.md` file
- add the status as a markdown file

#### How should this be manually tested?
- after raising a PR, check to see if it passed or failed

#### What are the relevant pivotal tracker stories?
#166315332

#### Questions:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Build Status
```
Comment (ebenezerdon) [inline README.md]: Good job @Chiagozielam.

Let's modify the file to contain this:

```
[![Build Status](https://travis-ci.org/newdevtech/newdev_api.svg?branch=develop)](https://travis-ci.org/newdevtech/newdev_api)
[![Coverage Status](https://coveralls.io/repos/github/newdevtech/newdev_api/badge.svg?branch=develop)](https://coveralls.io/github/newdevtech/newdev_api?branch=develop)
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[![Build Status](https://travis-ci.org/newdevtech/newdev_api.svg?branch=develop)](https://travis-ci.org/newdevtech/newdev_api)
+
```
Comment (ebenezerdon) [inline README.md]: Please remove this line.

Comment (Chiagozielam) [reply README.md]: Isn't that the build status?

Comment (ebenezerdon) [reply README.md]: The empty line, I mean.

Comment (ebenezerdon) [reply README.md]: We need to have the badges on the same horizontal line.

**Pull request merged**